code/modules/mob/dead/observer/orbit.dm

```diff
@@ -72,6 +73,8 @@
 			else if(M.stat == DEAD)
 				dead += list(serialized)
 			else
+				if(length(orbiters) >= 0.2 * length(get_observers())) // They're important if 20% of observers are watching them
```

Comment:
This outer loop already iterates over pretty much every mob, please call this `get_observers()` function before the loop.